src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs

```diff
@@ -648,6 +648,25 @@ public void TestParseTables_Negative(string script)
             TestParseErrors(script);
         }
 
+        [Theory]
+        [InlineData("a=10;")]
+        [InlineData("a=b=10;")]
+        [InlineData("a=10;c=20;")]
```

Comment:
Some functions can include the ; character in the function arguments. For example:

ClearCollect(A,B);If(A, Set(A, B) /*asdf*/;);Set(A, !B);

This formula was causing exceptions to be thrown because of a bug. Note that the If function call has a trailing ; before the close paren.

Reply:
Semicolons in the middle of expressions will be parsing errors.